tests/test_core.py

```diff
@@ -4117,6 +4117,11 @@ def test_mount(self):
     src = open(path_from_root('tests', 'fs', 'test_mount.c'), 'r').read()
     self.do_run(src, 'success', force_c=True)
 
+  def test_getdents64(self):
+    Settings.FORCE_FILESYSTEM = 1
```

Comment:
Some of the tests only use the FS from JS, and need forcing. For this test I don't see a reason it needs it. If it fails without it, let me know.